README.md

```diff
@@ -50,28 +50,33 @@ Please leave a comment that you're gonna work on it.
 #### 4. Create a new branch for your commits 
 We like to keep master branch clean so:
 ```bash
-# create a new branch and name it descriptively starting with number of your task e.g. "123_update_something":
-git checkout -b BRANCH NAME
+# create a new branch and name it descriptively starting with number of your task e.g. "123_UPDATE_SOMETHING":
+git checkout -b 123_BRANCH_NAME
 ```
 ```bash
 # Then, switch to your new branch that branch with command: 
-git checkout update_something
+git checkout 123_UPDATE_SOMETHING
 ```
 Now you can start developing and commiting your changes!
 
 #### 5. Push your changes to main repo
-Once you're finished first to commit your changes run three comannds. 
+Once you're finished, run four comanndsto commit your changes. 
 ```bash
-# First add yout changes to the commit: 
-git add .
+# First check your changes:
+git status
+# which will print the list of files you altered.
+```
+```bash
+#add changed files to your commit: 
+git add NAME/OF/THE.FILE
 ```
 ```bash
 # Then wrap the commit and describe what's happening there:
-git commit -m "description of changes"
+git commit -m "BRIEFLY DESCRIBE YOUR CHANGES"
```

Comment:
Same as above, I'd change it a bit:

`git commit -m "Briefly describe your changes"`

Reply:
Ditto